### DISCUSSION TITLE
Can I use Powered By Airbyte with OSS Version?

### DISCUSSION THREAD

**Discussion by nishit130** [airbytehq/airbyte#34196] [General]
Hey,
I have self-deployed airbyte on k8s cluster and wanted to use that airbyte instance for powered by airbyte offering.
I couldn't get the access credentials required for the PBA Api's. For eg [Initiate Oauth source](https://reference.airbyte.com/reference/initiateoauth) api requires access token for authentication how can I get it from my OSS deployed airbyte?

Any help would be highly appreciated,
Thanks

**Comment by sajarin** [comment]
Hey @nis130 

Sorry for the delay in response. For the time being, PbA is a cloud only feature. Feel free to reach out to our [team](https://airbyte.com/company/talk-to-sales) if you'd like to get set up with that particular offering or talk through the specifics of what your requirements look like.

[a comment by an automated account is not shown]
